.github/workflows/auto-update-setup-py.yaml

```diff
@@ -13,11 +11,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.head_ref }}
```

Comment:
Not sure this is needed? I'm pretty sure that's the default.

Reply:
We use the same ref to push the changes later: `git push origin ${{ github.head_ref }}`. Without this line the generated git command has a ref to PR number `/usr/bin/git checkout --progress --force refs/remotes/pull/1/merge` while with it the ref is to the branch: `/usr/bin/git checkout --progress --force -B update refs/remotes/origin/update`.

Reply:
got it!